vendor/src/github.com/docker/engine-api/types/auth.go

```diff
@@ -5,7 +5,6 @@ type AuthConfig struct {
 	Username      string `json:"username,omitempty"`
 	Password      string `json:"password,omitempty"`
 	Auth          string `json:"auth,omitempty"`
-	Email         string `json:"email"`
```

Comment:
I'm not sure it's a good idea to remove this field from the struct. It means that when docker saves the config file, it will strip out email addresses that were already present. Then it will no longer be possible to use that config file with an earlier version of the client, even if you don't touch any of the entries in question.

Maybe this is not a big concern to others. Thoughts?